tests/test_framework/revaultd.py

```diff
@@ -147,7 +148,7 @@ def stop(self, timeout=10):
 
     def cleanup(self):
         try:
-            self.stop()
+            self.stop
```

Comment:
??

Reply:
But you *don't call it*! If you don't put parenthesis you are not calling this function. If you have the error
> ''NoneType' object is not callable'

It means `stop` is `None`, which it should never be and you need to fix it.